### ISSUE TITLE
High contrast mode: 'Multiple images' icon not visible on screenshots thumbnail 

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7378]
'previous' and 'next' buttons in screenshot carousel are also not visible in high contrast mode.
